### PR TITLE
Hide project info by default

### PR DESCRIPTION
**Pull request by johanandren**
Also some styling to the table when visible

Preview:

Initial:
Screen Shot 2019-11-05 at 14 07 09

After toggle:
Screen Shot 2019-11-05 at 14 07 23

### CODE REVIEW

File: theme/src/main/assets/js/metadata-toggle.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+$(function() {
```
Comment (johanandren) [inline theme/src/main/assets/js/metadata-toggle.js]: on dom ready

File: theme/src/main/assets/js/metadata-toggle.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    var hiddenText = "[+] Show project info",
+        shownText = "[-] Hide project info",
+        toggle = $('<a href="#" class="project-info-toggle">' + hiddenText + '</a>'),
+        hidden = true,
+        infotable = $('table.project-info')
+
+    toggle.insertBefore(infotable)
```
Comment (johanandren) [inline theme/src/main/assets/js/metadata-toggle.js]: insert toggling link, table is hidden from start with css

**Comment by jrudolph** [comment]
I'm not opposed to the change but I wonder if the whole panel isn't a bit useless if it isn't shown at all. What would be the developer workflow where someone is looking at the documentation and voluntarily click that link?

**Comment by johanandren** [comment]
@jrudolph I agree it is questionable let's discuss that later elsewhere and just make sure 2.6.0 docs looks good for now.

**Comment by jrudolph** [comment]
> @jrudolph I agree it is questionable let's discuss that later elsewhere and just make sure 2.6.0 docs looks good for now.

Yep, sounds reasonable.

**Comment by patriknw** [comment]
I see it mostly as a way to have the information in a consistent way (not sure if anyone will every look at it, and therefore it shouldn't take up an entire page)

**Pull request merged**